src/Infrastructure/LeanCode.ConfigCat/ConfigCatExtensions.cs

```diff
@@ -0,0 +1,128 @@
+using System.Text.Json;
+using ConfigCat.Client;
+using LeanCode.ConfigCat;
+using Microsoft.Extensions.DependencyInjection.Extensions;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+
+[assembly: System.Diagnostics.CodeAnalysis.SuppressMessage(
```

Comment:
I wonder if we should move (not now, but sometime soon) all the extensions to `IServiceCollection` to this namespace.

Reply:
I believe we should :slightly_smiling_face: